### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by yuanpengtu** [pleaseconnectwifi/PNAL#10]
Hi, it seems that the code takes a long time to train when using the default batch size. But when I try to increase the batch size and learning rate to accelerate the process, the performance of the algorithm deteriorates significantly. Are there other ways to speed up the training process？
